apps/onnx/benchmarking_utils.h

```diff
@@ -0,0 +1,21 @@
+#ifndef BENCHMARKING_UTILS_H_
+#define BENCHMARKING_UTILS_H_
+
+class CacheEvictor {
```

Comment:
I don't understand the purpose of this class. We update buffer but never do anything with it.

Reply:
I am simply flushing the content of all the CPU caches by updating more data than what would fit in cache. This is simply needed when benchmarking performance in order to get more reliable performance numbers

Reply:
Done